run_microbenchmarks.py

```diff
@@ -0,0 +1,25 @@
+import argparse
+from torchbenchmark.microbenchmarks import get_nvfuser_microbenchmarks
+
+
+def run():
+    parser = argparse.ArgumentParser(description="Run nvfuser microbenchmarks")
+    parser.add_argument("--filter", nargs="*", default=[], help='List of benchmarks to test')
+    parser.add_argument("--fusers", nargs="*", default=[], help='List of fusers to run tests on (options include "no_fuser", "fuser0", "fuser1", "fuser2")')
```

Comment:
You can use `choices=['no_fuser', 'fuser0', 'fuser1', 'fuser2']` to specify possible options.